drivers/soundwire/stream.c

```diff
@@ -1693,6 +1692,7 @@ static int _sdw_deprepare_stream(struct sdw_stream_runtime *stream)
 				return ret;
 			}
 		}
+		bus->params.bandwidth -= bandwidth - multi_lane_bandwidth;
```

Comment:
not able to follow the commit message:
a) typo on "the bandwidth will of the stream" makes it unclear
b) "when compute_params() is called in sdw_prepare_stream()" The function modified is sdw_deprepare_stream, not sure why there's a reference to the prepare step.
c) it's not clear if the code is just a move or if bus->params.bandwidth can change inside of bus->compute_params().

Reply:
The issue is that I will get "Compute transport params failed: -22" error without this commit. The issue is that we count bandwidth of all m_rt including the stream that is depreparing in bus->compute_params()->sdw_compute_port_params(). If we subtract bandwidth before calling bus->compute_params(), sdw_compute_bus_params() will select the clock that doesn't count the bandwidth of the stream that is depreparing.
To answer your questions.
a) That should be "the bandwidth of the stream"
b) Sorry, that was a typo, too. It should be "when compute_params() is called in sdw_deprepare_stream()"
c) bus->params.bandwidth will not be changed before multilane support added. However, bus->params.bandwidth will be modified when multilane is used.